components/layout/display_list/items.rs

```diff
@@ -935,14 +943,17 @@ impl fmt::Debug for DisplayItem {
             return write!(f, "DefineClipScrollNode({:?}", item.node_index);
         }
 
-        write!(f, "{} @ {:?} {:?}",
+        write!(
+            f,
+            "{} @ {:?} {:?}",
             match *self {
-                DisplayItem::SolidColor(ref solid_color) =>
-                    format!("SolidColor rgba({}, {}, {}, {})",
-                            solid_color.color.r,
-                            solid_color.color.g,
-                            solid_color.color.b,
-                            solid_color.color.a),
+                DisplayItem::SolidColor(ref solid_color) => format!(
+                    "SolidColor rgba({}, {}, {}, {})",
```

Comment:
Same for these, I'm fine with formatting changes, but they should be a different commit.